testes/módulo-separado/factory/makeStyleTableBasedOnStyleName.js

```diff
@@ -0,0 +1,67 @@
+// Author: Felipe Barros <felipetech@tuta.io>
+// 22 September 2020
+
+import makeElement from './makeElement.js';
+import getImageLink from '../functions/getImageLink.js';
+import getColorsBasedOnStyleName from '../functions/getColorsBasedOnStyleName.js';
+import getCategoriesBasedOnStyleName from '../functions/getCategoriesBasedOnStyleName.js';
+
+/**
+ * @param {string} styleName
+ */
+function makeStyleTableBasedOnStyleName(styleName) {
+  const table = makeElement('table');
+  table.setAttribute('id', styleName);
+
+  const thead = makeElement('thead');
+  const tbody = makeElement('tbody');
+
+  const trHeader = makeElement('tr');
+  const firstHeader = makeElement('th');
+  const secondHeader = makeElement('th');
```

Comment:
Por favor, expande a opção, @kazordoon:

```suggestion
  const firstHeader = makeElement('th', {class: 'c'});
  const secondHeader = makeElement('th', {class: 'c'});
```